src/Nethermind/Nethermind.Baseline.Test/BaselineConfigTests.cs

```diff
@@ -29,11 +29,37 @@ public class BaselineConfigTests : ConfigFileTestsBase
         public void Can_set()
         {
             BaselineConfig config = new BaselineConfig();
+            var dbCacheIndexAndFilterBlocks = true;
+            uint dbBlockCacheSize = 100;
+            uint dbWriteBufferSize = 300;
+            uint dbWriteBufferNumber = 3;
             config.Enabled.Should().BeFalse();
             config.Enabled = true;
             config.Enabled.Should().BeTrue();
             config.Enabled = false;
             config.Enabled.Should().BeFalse();
+
+            config.BaselineTreeDbCacheIndexAndFilterBlocks.Should().BeFalse();
```

Comment:
can we get rid of this and hardcode it / read it - I do not want user to have to set this up